### ISSUE TITLE
problem with astroid menu

### ISSUE THREAD

**Issue by SerGun78** [templaza/astroid-framework#421]
Hello Sonny! These three settings don't work:
![изображение](https://github.com/templaza/astroid-framework/assets/95705148/a1051f78-4020-4640-86ce-107b334f2787)

You also need to add a choice of the location of menu links: either horizontal or vertical

**Comment by Joomlaplates** [comment]
Hi, the main menu settings can be done in Astroid Template settings and not in the module itself.
And for sidebar or footer menu you dont need the Astroid menu, just take the basic menu typ and chose the layout.
Regards Peter
![Screenshot_1](https://github.com/templaza/astroid-framework/assets/71080509/b4d0d3f1-7af8-4a8c-b72e-942f5aabfabe)

**Comment by SerGun78** [comment]
Thanks, I know about this setting. The question then is: why did they introduce the “astroid menu” module into the package back then? what is it for?

**Comment by Joomlaplates** [comment]
Hello, you dont need the Astroid menu modul.
Why its available? I dont know, 
Greatings Peter

**Comment by SerGun78** [comment]
this module was introduced in Astroid Framework Version 2.6.6.
I hope Sonny can explain why it is needed and why the above settings do not work in it.

**Comment by Chacapamac** [comment]
This module is to be able to have another menu that will look exactly like the main menu based on Astroid classes and all.

**Comment by SerGun78** [comment]
In the current version, this version of the astroid module is inferior, because does not have all the settings that are in the one/zero templates

**Comment by SerGun78** [comment]
In the current version, this version of the astroid module is inferior, because does not have all the settings that are in the one/zero templates. at the moment this is the only way it works:
![изображение](https://github.com/templaza/astroid-framework/assets/95705148/f2cf553d-05bc-4e39-8b7a-77b368d8d380)

and where is the view for the sidebar as in the template settings?
![изображение](https://github.com/templaza/astroid-framework/assets/95705148/8298e251-9801-4012-892e-3384d020b8ad)

**Comment by SerGun78** [comment]
No. I want to display both the Horizontal Menu in the template and the side menu, as you can select in the one/zero template.
and to have styles css and fonts like in the main menu in headers, you need to use the "astroid menu" module.

**Comment by Chacapamac** [comment]
@SerGun78 

> this is the only way it works:

On my test website, at the bottom you can see a repeat of the main menu made with that module.
http://joom4-astro.graf-test.ca/
![Screen Shot 2023-10-26 at 10 21 17 AM](https://github.com/templaza/astroid-framework/assets/1907276/18b6cbf4-e108-4544-92b0-78d8133771af)

**Were you able to achieve what you want?**

**Comment by SerGun78** [comment]
It is necessary that in the astroid menu module you can select its type, as in the one/zero templates. in theory there are three of them, but in fact there are two: Horizontal=Stacked (since their functionality and appearance are the same) and Sidebar (which has a drop-down list of menu sub-items with “+” and “-”).
Why not implement something like this in the astroid menu module?
There is no need for such a menu in the footer, as in the header.

**Comment by Chacapamac** [comment]
@SerGun78 , Yes this Astroid Menu Module can probably be better with your great suggestions

> There is no need for such a menu in the footer, as in the header.
Yes obviously, this is only a demo site to test Astroid functions, obviously, the menu can be whatever you want.

I will do a test in the sidebar to see how it go.

**Comment by felixkat** [comment]
Closing this issue but have referenced it here  https://github.com/templaza/astroid-framework/issues/814
